crml/cennzx-spot/src/lib.rs

```diff
@@ -30,10 +30,27 @@ extern crate srml_support as support;
 
 use generic_asset;
 use rstd::prelude::*;
-use runtime_primitives::traits::{As, Bounded, One, Zero};
-use support::{dispatch::Result, Dispatchable, Parameter, StorageDoubleMap, StorageMap, StorageValue};
+use runtime_primitives::traits::{Bounded, One, Zero};
+use support::{
+	dispatch::Result as DispatchResult, Dispatchable, Parameter, StorageDoubleMap, StorageMap, StorageValue,
+};
 use system::ensure_signed;
 
+// TODO: remove this temp fix for overflow issue when upstream fix ready
+#[macro_use]
+extern crate uint;
+use core::convert::TryInto;
+
+construct_uint! {
```

Comment:
should just use the one defined in `types.rs`

Reply:
done